src/enums/nameType.ts

```diff
@@ -0,0 +1,4 @@
+export enum NameType {
```

Comment:
I wonder if it's worth it moving this enum to the shared components. I'm probably going to have it in Edit UI as well. Thoughts?

Reply:
Using the shared enum now!